derive_where attribute macro instead of derive macro (#33)
ModProg/derive-where: README.md

[derivative](https://crates.io/crates/derivative)
([![Crates.io](https://img.shields.io/crates/v/derivative.svg)](https://crates.io/crates/derivative))

> is a great alternative with many options. Notably it has no `no_std`
> support and requires to `#[derive(Derivative)]` to use.

Reviewer: ```suggestion
is a great alternative with many options. Notably it doesn't support `no_std`
and requires an extra `#[derive(Derivative)]` to use.
```

While are here, too bad none of us are native speakers :smile:.
